lib/ares.rb

```diff
@@ -4,6 +4,8 @@
 require 'ares/errors'
 require 'ares/logging'
 require 'ares/client'
+require 'rails'
+require 'ico-validator'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.